doc/source/user/building.rst

```diff
@@ -159,6 +163,12 @@ will prefer to use ATLAS, then OpenBLAS and as a last resort MKL.
 If neither of these exists the build will fail (names are compared
 lower case).
 
+The native libraries on macOS, provided by Accelerate, are not fit for use
+in NumPy since they have bugs that cause wrong output under easily reproducable
+conditions. If the vendor fixes those bugs, the library could be reinstated,
+but until then users compiling for themselves should use another linear
+library or use the built-in (but slower) default, see the next section. 
+
```

Comment:
@charris: is something like this what you had in mind?

Reply:
Yes. I'd probably put an asterisk on Accelerate and make this a footnote (or equivalent)

Reply:
But at this point, NumPy doesn't support Accelerate, correct? So maybe take it off the lists and keep the comment with a reference to the version in which accelerate was removed.

Reply:
Added a deprecation directive. Is [this](https://13956-908607-gh.circle-artifacts.com/0/doc/build/html/user/building.html#lapack) what you meant? Should I move that whole paragraph to the top/bottom of the section?

Reply:
@charris: see above comment